Backpack/src/main/java/net/skyscanner/backpack/button/internal/ButtonStyle.kt

```diff
@@ -37,12 +38,18 @@ internal class ButtonStyle(
   @ColorInt private val bgColor: Int,
   @ColorInt private val bgPressedColor: Int,
   @ColorInt private val bgDisabledColor: Int,
-  private val contentColor: ColorStateList,
+  @ColorInt private val contentColor: Int,
+  @ColorInt private val contentPressedColor: Int,
+  @ColorInt private val contentDisabledColor: Int,
   @ColorInt private val rippleColor: Int,
 ) {
 
   fun getContentColor(): ColorStateList =
-    contentColor
+    colorStateList(
```

Comment:
Basically, this is the bugfix for icon alignment. It fixes when you switch to custom ColorStateList creation instead of fetching from resources.

Reply:
that is so weird! any idea why this is happening?

Reply:
Resources has an internal cache for CST, it may be related. But no, I've figured it out by filtering the changes.

Reply:
Indded